icons/cloth.svg

```diff
@@ -0,0 +1,13 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M5.3 9.7c.4-.4.4-1 0-1.4L3.7 6.7c-.4-.4-.4-1 0-1.4l1.6-1.6c.4-.4 1-.4 1.4 0l1.6 1.6c.4.4 1 .4 1.4 0l1.6-1.6c.4-.4 1-.4 1.4 0l1.6 1.6c.4.4 1 .4 1.4 0l1.6-1.6c.4-.4 1-.4 1.4 0l1.6 1.6c.4.4.4 1 0 1.4l-1.6 1.6c-.4.4-.4 1 0 1.4l1.6 1.6c.4.4.4 1 0 1.4l-1.6 1.6c-.4.4-.4 1 0 1.4l1.6 1.6c.4.4.4 1 0 1.4l-1.6 1.6c-.4.4-1 .4-1.4 0l-1.6-1.6c-.4-.4-1-.4-1.4 0l-1.6 1.6c-.4.4-1 .4-1.4 0l-1.6-1.6c-.4-.4-1-.4-1.4 0l-1.6 1.6c-.4.4-1 .4-1.4 0l-1.6-1.6c-.4-.4-.4-1 0-1.4l1.6-1.6c.4-.4.4-1 0-1.4l-1.6-1.6c-.4-.4-.4-1 0-1.4l1.6-1.6z" />
```

Comment:
Big fan of this one, but it might need to be a bit bigger?

Reply:
> it might need to be a bit bigger?

@jguddas `push`ed…